packages/wasm_coq/wasm_coq.0.1/opam

```diff
@@ -0,0 +1,47 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Wasm formalisation in Coq"
+description:
+  "Wasm formalisation in Coq, following the AFP formalisation of Conrad Watt"
+maintainer: ["Xiaojia Rao" "Martin Bodin"]
+authors: [
+  "Martin Bodin" "Philippa Gardner" "Jean Pichon" "Xiaojia Rao" "Conrad Watt"
+]
+license: "MIT"
+homepage: "https://github.com/WasmCert/WasmCert-Coq"
+bug-reports: "https://github.com/WasmCert/WasmCert-Coq/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "coq" {>= "8.16" & <= "8.17.1"}
+  "coq-compcert" {>= "3.11"}
```

Comment:
This package doesn't appear to be in the opam-repository; is it in the upstream coq one perhaps? It makes this package uninstallable here.

Reply:
Ah this might be an issue -- CompCert is only in the Coq repository indeed. Is there an easy way to fix this on the main opam repository or should I maybe look to publish it on the Coq repository instead?

Reply:
I think I figured it out -- the Coq repo might be the place instead (https://github.com/coq/opam).

Thanks for the message!